src/gc/bits.d

```diff
@@ -19,14 +19,21 @@ import core.stdc.string;
 import core.stdc.stdlib;
 import core.exception : onOutOfMemoryError;
 
+
+// use version bitwise to disable optimizations that use word operands
+// on bulk operation copyRange, setRange, clrRange, etc.
+// version = bitwise;
```

Comment:
That version should probably be namespaced and more descriptive.

Reply:
Not sure what you mean. `version = gc_bits_bitwise;`?